tests/integ/modin/test_apply_snowpark_python_functions.py

```diff
@@ -89,3 +89,14 @@ def test_apply_snowflake_cortex_summarize():
     summary = s.apply(snowflake_cortex_summarize).iloc[0]
     # this length check is to get around the fact that this function may not be deterministic
     assert 0 < len(summary) < len(content)
+
+
+@sql_count_checker(query_count=1)
+@pytest.mark.skip("SNOW-1758914 snowflake.cortex.sentiment error on GCP")
+def test_apply_snowflake_cortex_sentiment():
+    from snowflake.snowpark.functions import snowflake_cortex_sentiment
+
+    content = "A very very bad review!"
+    s = pd.Series([content])
+    sentiment = s.apply(snowflake_cortex_sentiment).iloc[0]
+    assert -1 <= sentiment <= 1
```

Comment:
Is it not safe to make the boundaries here between -1 and 0 instead of -1 and 1?

Reply:
I realize that the return value may be slightly different each time we call the function. But I'd expect it to remain on the negative side for this particular review, no?

Reply:
That is true, if the cortex function works as expected this should be between -1 and 0. If we want to evaluate the validity of the cortex function itself, I can update this!

Reply:
Yes, I think it will make the test case more meaningful.

Reply:
Sounds good, I've updated the sentiment tests!